content_script.js

```diff
@@ -104,14 +104,29 @@
           clipboardItems['text/rtf'] = new Blob(
             [
               `{\field{\*\fldinst HYPERLINK "${url}"}{\fldrslt ${
-                  selectedText}}}`,
+                  selection.toString()}}}`,
             ],
             {
               type: 'text/rtf',
             }
           );
           */
         }
+        else if (linkStyle === 'html') {
+          var html = "";
+          if (selection.rangeCount) {
+              var container = document.createElement("div");
+              for (var i = 0, len = selection.rangeCount; i < len; ++i) {
+                  container.appendChild(selection.getRangeAt(i).cloneContents());
+              }
+              html = container.innerHTML;
+          }
+          clipboardItems['text/html'] = new Blob(
+              [`${html} [🔗](${url})`],
```

Comment:
This looks like Markdown to me, not HTML.

Reply:
Explained here in the first comment:

> I noticed that the URL in the rich style has some encoding issue: the spaces are still space which should be %20. Didn't figure out a way to fix it. So in the HTML style, the link is added as a markdown style.

(I am not sure if someone has reported the URL encoding issue before...)

Reply:
Thanks. Could you then please fix the encoding issue so a proper HTML link can be used. See https://github.com/GoogleChromeLabs/text-fragments-polyfill/issues/115.

Reply:
Looks like this has been fixed in e0b0a85c8a169da67a09752abd2d992cbdeac7fc